src/utils/url.ts

```diff
@@ -199,3 +199,7 @@ export function isURLEnabled(url: string, userSettings: UserSettings, {isProtect
     }
     return (!isInDarkList && !isURLInUserList);
 }
+
+export function isFullyQualifiedDomain(candidate: string) {
+    return /^[a-z0-9.-]+$/.test(candidate);
```

Comment:
Not sure if it matters but this could be turned into a anti-pattern with simply: `/^(\*|\^|\$)$/` as in terms of regex it would be faster to only loop once trough the text(without tracking the state of the previous character) and check if such character exists.

Reply:
I created a rudimentary microbenchmark and get slightly better results with `/^[a-z0-9.-]+$/` than with `/^(\*|\^|\$)$/`. Will create a proper benchmark and share result by the end of today.

Reply:
Sure things, even though I think it might be more efficient to just do it like: `/[\*\^\$]/` 🤷🏽‍♂️

Reply:
![image](https://user-images.githubusercontent.com/25481501/131547310-7d81cb28-6e01-4820-b1ab-6d5c7df57866.png)
Seems like the current one is indeed the fastest, so let it be as is.